### PR TITLE
Fixed typo in services build.gradle resulting in directions refresh not being included in services artifact

### PR DESCRIPTION
**Pull request by devotaaabel**

### CODE REVIEW

File: services-directions/src/main/java/com/mapbox/api/directions/v5/models/DirectionsRoute.java [hunk 1]
```diff
@@ -39,7 +39,7 @@ public static Builder builder() {
    * @since 4.4.0
    */
   @Nullable
-  public abstract String routeIndex();
+  public abstract Integer routeIndex();
```
Comment (Guardiola31337) [inline services-directions/src/main/java/com/mapbox/api/directions/v5/models/DirectionsRoute.java]: This is breaking SemVer and it seems out of scope of this PR.

Comment (osana) [reply services-directions/src/main/java/com/mapbox/api/directions/v5/models/DirectionsRoute.java]: Good point!
